hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2498,6 +2482,7 @@ class Emit[C](
               cb.assign(stageName, stageName.concat("|").concat(dynamicID.asString.loadString(cb)))
             })
 
+          val encRes = cb.newLocal[Array[Array[Byte]]]("encRes")
           cb.assign(encRes, spark.invoke[BackendContext, HailClassLoader, FS, String, Array[Array[Byte]], Array[Byte], String, Option[TableStageDependency], Array[Array[Byte]]](
```

Comment:
Approved, but is it possible to use `memoize` here as well or is this special in some way?